src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx

```diff
@@ -213,4 +213,19 @@
   <data name="DatabaseErrorPage_NoMatch" xml:space="preserve">
     <value>The current exception (and its inner exceptions) do not match the last exception Entity Framework recorded due to a failed database operation. This means the database operation exception was handled and another exception occurred later in the request.</value>
   </data>
+  <data name="DatabaseErrorPage_AddMigrationCommandPMC" xml:space="preserve">
+    <value>PM&gt; Add-Migration [migration name]</value>
+  </data>
+  <data name="DatabaseErrorPage_ApplyMigrationsCommandPMC" xml:space="preserve">
+    <value>PM&gt; Update-Database</value>
+  </data>
+  <data name="DatabaseErrorPage_NoDbOrMigrationsInfoCLI" xml:space="preserve">
+    <value>Alternatively, you can scaffold a new migration and apply it from a command prompt at your project directory:</value>
+  </data>
+  <data name="DatabaseErrorPage_PendingChangesInfoCLI" xml:space="preserve">
+    <value>Alternatively, you can scaffold a new migration and apply it from a command prompt at your project directory:</value>
```

Comment:
It is, they are used in different places on the page, so I kept them separate to make it easier to adjust the strings in the future.